src/server/common/model/answer/answer-model.js

```diff
@@ -67,6 +68,11 @@ export class AnswerModel {
     throw new NotImplementedError()
   }
 
+  /** @returns {Page} */
+  nextPage() {
+    throw new NotImplementedError()
+  }
```

Comment:
The answer models are intended to be the more re-usable part - e.g. Address, Email address or name being generic - so the nextPage is on the Page abstraction as a result.